index.html

```diff
@@ -506,6 +506,18 @@ <h1>
 <a href="#extensibility"></a>.
     </p>
 
+    <section>
+      <h2>
+Identifier
+      </h2>
+      <p>
+Identifiers are used in the data model to identify specific instances of people,
+keys, organizations, services, entities, and things in general. Identifiers are
```

Comment:
Devices, too?

Reply:
The rest looks good to me.

Reply:
Perhaps `specific instances of classes, including people` so that we don't have to enumerate *every* class (such as devices).

Reply:
I think "instance of class" is too techy... I can replace entities with devices to keep the list short.

Reply:
Fixed in f0912558175e69ce95834d5d37d2db3521482e0b.